pkg/arvo/sys/vane/gall.hoon

```diff
@@ -1906,6 +2053,21 @@
   =/  =coin  $/r.bem
   =*  path  s.bem
   ::
+  ?:  ?=(%a care)
+    ?.  =(p.bem our)  ~
+    =/  yok  (~(get by yokes.state) q.bem)
+    ?~  yok  ~
+    =/  ski  (~(get by sky.u.yok) s.bem)
+    ?~  ski  ~
+    =/  res=(unit (each noun @uvI))  
+      ?+  -.r.bem  ~
+        %ud  (get:on-path fan.u.ski p.r.bem)
+        %da  ?>(=(p.r.bem now) (bind (ram:on-path fan.u.ski) tail))
+      ==
+    ?.  ?=([~ %& *] res)  ~
+    ``[%noun %noun p.u.res]
+  ::  TODO new care to ask for hash of a sky binding
+  ::
```

Comment:
It'd be great to use %z for this hash, and to also support %y for `arch`

Reply:
%y is coming!

Reply:
%y is interesting because the aeons of different paths may not be in sync with each other. Say that we have these paths bound:
```
/==/0/foo
/==/1/foo
/==/0/foo/bar
```
What should happen if we scry for `gy//==/1/foo`? Should we get `[[~ <foo-hash>] ~]` or `[[~ <foo-hash>] (malt [~.bar ~]~)]`?

An alternative solution would be to force all aeons within the same agent to be in sync, and assume that data bound at one aeon is also bound for all other aeons until the next bound aeon on that same path. Feels icky to layer that sort of interpretation on top of the bindings though.

Reply:
Sounds good, `%t` is implemented!